pkgs/top-level/darwin-packages.nix

```
@@ -234,14 +234,20 @@ impure-cmds // appleSourcePackages // chooseLibs // {
           ] ++ modules;
 
           virtualisation.host = { inherit pkgs; };
```

can you also port this change from the other pr?
```suggestion
          virtualisation.host.pkgs = lib.mkOverride 900 pkgs;
```

---

What's the reason for this change? If someone really wants to override it, I think `lib.mkForce` still works

---

If they want a different host packages, they shouldn't be starting out with the wrong host packages in the first place.
Instead of `pkgs.darwin.linux-builder`, they should call `someOtherPkgs.darwin.linux-builder`.
This line does not apply to the general module, but only to the package.

Indeed they could `lib.mkForce` as a last resort, but they should really just clean up their code by either using the right Nixpkgs in the first place, or by using the module directly, which also bypasses this.